### DISCUSSION TITLE
Notifying all subscribers of a topic is not working properly

### DISCUSSION THREAD

**Discussion by RuslanHryn** [magicbell/community#213] [Questions] [answered]
I tried to notify all subscribers of a topic using `/broadcasts` endpoint and HTTP client that generated from Open API spec
However, I was able to send notifications using emails or externalId.
I can't notify all subscribers of a topic using API.

I got the following error `{"errors":[{"message":"Param 'broadcast.recipients.0' doesn't allow key: topic"},{"message":"Param 'email' is missing"}]}`  if set `broadcast.topic` and `recipients.topic`

What is the proposal for having a topic in multiple fields?
```
{
  "broadcast": {
    "title": "test",
    "content": "test",
    "category": "order_created",
    "topic": "order:33098",
    "recipients": [
      {
        "topic": "order:33098"
      }
    ]
  }
}
```

if set only `broadcast.topic` - `{"errors":[{"message":"Param 'broadcast.recipients' must contain atleast 1 items"}]}`
```
{
  "broadcast": {
    "title": "test",
    "content": "test",
    "category": "order_created",
    "topic": "order:33098"
  }
}
```

if set only `recipients.topic` -  `{"errors":[{"message":"Param 'broadcast.recipients.0' doesn't allow key: topic"},{"message":"Param 'email' is missing"}]}`

```
{
  "broadcast": {
    "title": "test",
    "content": "test",
    "category": "order_created",
    "recipients": [
      {
        "topic": "order:33098"
      }
    ]
  }
}
```

I was able to send notification when specified externalId. In this case only single user received notification but other subscribers did not receive any notifications

```
{
  "broadcast": {
    "title": "test",
    "content": "test",
    "category": "order_created",
    "topic": "order:33098",
    "recipients": [
      {
        "externalId": "123456"
      }
    ]
  }
}
```

So how to send notification to all subscribers?

**Comment by unamashana** [comment, marked as answer]
Hey @RuslanHryn please try this syntax for the topic recipients 

```
"recipients": [
      {
        "topic": {
			"subscribers": true
		}
      }
    ]
```

**Comment by RuslanHryn** [comment]
@unamashana thanks. It is working properly.

Are there any additional values from the topic field?
